### PR TITLE
Add readme.txt for WordPress.org plugin repo

### PR DESCRIPTION
**Pull request by hajekj**
#186

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+### 3. Configure the plugin
+
+Once the plugin is activated, update your settings from the WordPress admin console under **Settings** > **Azure AD**. Basic settings to include are:
+**Enable Azure AD group to WP role association** - Check this to enable Azure AD group-based WordPress roles.
+**Default WordPress role if not in Azure AD group** - This is the default role that users will be assigned to if matching Azure AD group to WordPress roles is enabled. If this is not set, and the user authenticating does not belong to any of the groups defined, they will be denied access.
+**WordPress role to Azure AD group map** - For each of the blog's WordPress roles, there is a field for the ObjectId of the Azure AD group that will be associated with that role.
+
+== Frequently Asked Questions ==
+For more configuration information and bug reports, please visit [plugin's repo on GitHub](https://github.com/hajekj/aad-sso-wordpress).
```
Comment (psignoret) [inline readme.txt]: Can you please change to https://github.com/psignoret/aad-sso-wordpress?

File: readme.txt [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+Contributors: psignoret, hajekj
+Tags: azure-ad, sso, azure-active-directory
+Requires at least: 4.1
+Tested up to: 4.9.5
+Requires PHP: 5.6
+Stable tag: 0.6.4
+License: MIT
+License URI: https://github.com/hajekj/aad-sso-wordpress/blob/master/LICENSE.md
+
+Allows you to use your organization\'s Azure Active Directory user accounts to log in to WordPress. If your organization is using Office 365, your user accounts are already in Azure Active Directory. This plugin uses OAuth 2.0 to authenticate users, and the Azure Active Directory Graph to get group membership and other details.
```
Comment (psignoret) [inline readme.txt]: Should be less than 150 words. Proposed: "Sign in to WordPress using your organization's Azure Active Directory accounts (the same ones used to sign in to Office 365)."

File: readme.txt [hunk 3]
```diff
@@ -0,0 +1,82 @@
+=== Azure AD Single-Sign-On ===
+Contributors: psignoret, hajekj
```
Comment (psignoret) [inline readme.txt]: Please also include @bradkovach @ryanwelcher @blobaugh @jtsternberg and @christianhennen.

Comment (psignoret) [reply readme.txt]: (As far as I can tell, those are all also their WordPress.org usernames.)

File: readme.txt [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+In the typical flow:
+
+1. User attempts to log in to the blog (wp-admin). At the sign in page, they are given a link to sign in with their Azure Active Directory organization account (e.g. an Office 365 account).
+2. After signing in, the user is redirected back to the blog with an authorization code, which the plugin exchanges for a ID Token, containing a minimal set of claims about the signed in user, and an Access Token, which can be used to query Azure AD for additional details.
+3. The plugin uses the claims in the ID Token to attempt to find a WordPress user with an email address or login name that matches the Azure AD user.
+4. If one is found, the user is authenticated in WordPress as that user. If one is not found, the WordPress user will (optionally) be auto-provisioned on-the-fly.
+5. (Optional) Membership to certain groups in Azure AD can be mapped to roles in WordPress, and group membership can be used to restrict access.
+
+== Installation ==
+### 1. Download and activate the plugin
```
Comment (psignoret) [inline readme.txt]: This first step should reference installation through WordPress.org plugin directory.

**Comment by hajekj** [comment]
I will update the PR momentarily with the fixes. Sorry for the URL, copied the wrong repo address.

**Comment by hajekj** [comment]
I have made the requested changes.

**Pull request merged**